examples/tomo/checks/check_axes_cone2d_bp.py

```diff
@@ -0,0 +1,59 @@
+"""Cone beam 2D example for checking that orientations are handled correctly.
+
+Due to differing axis conventions between ODL and the ray transform
+back-ends, a check is needed to confirm that the translation steps are
+done correctly.
+
+The back-projected data should be a blurry version of the phantom, with
+all features in the correct positions, not flipped or rotated.
+
+This example is best run in Spyder section-by-section (CTRL-Enter).
+"""
+
+# %% Set up the things that never change
+
+import numpy as np
+import odl
+
+# Set back-end here
+impl = 'astra_cuda'
+# Set a volume shift. This should not have any influence on the back-projected
+# data.
+shift = (0, 25)
+
+img_shape = (100, 150)
+img_max_pt = np.array(img_shape, dtype=float) / 2
+img_min_pt = -img_max_pt
+reco_space = odl.uniform_discr(img_min_pt + shift, img_max_pt + shift,
+                               img_shape, dtype='float32')
+phantom = odl.phantom.indicate_proj_axis(reco_space)
+
+assert np.allclose(reco_space.cell_sides, 1)
+
+# Take 1 degree increments, full angular range
+grid = odl.RectGrid(np.linspace(0, 2 * np.pi, 360, endpoint=False))
+angle_partition = odl.uniform_partition_fromgrid(grid)
```

Comment:
Well it's not quite the same. Here it wouldn't matter much, but if the angles used for the actual projections (and back-projections) should be exactly as specified, one has to do this, or more simply, use `odl.nonuniform_partition`.
Here I can change it to simple, but we should think about the other examples where we kind of instruct people in a slightly wrong way.